templates/admin/base_site.html

```diff
@@ -18,4 +18,8 @@
 {% block extrastyle %}
     <link rel="stylesheet" type="text/css" href="{% static "css/local.css" %}" />
     <link rel="stylesheet" type="text/css" href="{% static "css/test-banner.css" %}" />
+
+    <script> {# make sure jquery is available #}
+    if (!$) { $ = django.jQuery;}
```

Comment:
doesn't this need `var`, `let`, or `const` in order to be a valid js declaration?

Reply:
oh nvm, it's defining on `window` probably

Reply:
I borrowed this from some django answers on stack overflow or similar. Seemed to work for what I needed.